### PR TITLE
Reduces Pilot Officer slots to 1

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request
Title.
Renames role to CAS Pilot Officer.
## Why It's Good For The Game
Reduces role bloat.

Pilot Officer's only value is the access they have and would be better if they focused solely on CAS. This will help encourage other roles like FC/SL/CSE or even Synth to use tadpole to support marines in various ways and not relegate a player that's the equivalent to SL- stuck on tadpole duty.
## Changelog
:cl:
balance: Reduces available PO slots to 1.
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
this is pretty bad

forcing pos, or, po in this case, to only play cas would most likely just increase suicide rates (number of people playing po) to drop significantly because as it stands, Cas feels incredibly unrewarding to play, only with the occasional kills here and there and moral support from marines groundside which almost never happens, mortar or howie is way better at cas's job than Cas itself. 
Taking FC, SLS, and synths out of their usual roles to play tad instead of leading, advising and being logistics feels like hampering their abilities when two posdid the job before just fine for em worked before and certainly ain't broken to warrant them being reduced now.

The only role I do see somewhat fitting to what you just said is CSE, but I think because out of all the roles they basically are just CL in an engineering costume, fluff, with no actual role attributed to them and the ones they do have is reloading either gens, or OB, which is pretty trivial. Plus CSE can already tag along with POs on tadpole before if they want to be tad crew. Maybe a CSE specific PR that retouches their role simmilar to how researcher got a rework from being basement dwelling nerd to xeno prober master? 

All in all this seems like it hampers more than it benefits both the pos and the roles mentioned above, forces the singular po to only play cas severely cutting back the versatility of the role and the fun/enjoyment of the player and is rather unnecessary.

if the aim of this pr was to give the already limited things pos can do to other roles who are far better of doing their own... roles, then I guess it succeeds in that aspect.

**Comment by Coastfront** [comment]
Ignoring the fact that this would then divert important roles to have to play tad, leaving a hole where they left with nobody to fill it, this is pointless.

If a new player winds up on tad, sucks to suck currently and thats just how it is. Forcing PO's to play CAS just removes the dynamic part of their role that they can fufill different varities of duties in different ways.
Even as some people treat PO as "the tad role", redirecting other important roles to play as "the tad role" (synth, SL, FC, etc) is using one puzzle piece to fill the hole of another: theres still a blank space left.

**Comment by KerbalCitizen** [comment]
Even if synth, SL, CSE, or FC are using the tad, tad PO is still nice to have. Tad PO can stay on the tad to ensure it won't die, fighting off pressure with mounted hsg 102, or just taking off if too much pressure on it. Meanwhile, the synth, SL, CSE, or FC can leave the tad knowing its in good hands and that it won't be lost. I've worked with CSE Derrick a few times as tad PO. It's nice that he can go out and fix an APC, where he could easily be killed, while tad still remains safe under my supervision. 

Not a fan of this change at all personally.

**Comment by Lumipharon** [comment]
Non PO pilots have other things to do off the tad, PO should not.

I don't really see the point of this, if you want to do dedicated tad stuff, be a PO.

**Comment by Al-1ce** [comment]
I've got around 90 hours on PO, which is probably more than everybody else commenting on this PR combined.

I can say with confidence that this is for the better. Having to deal with another other pilot wanting to operate the same thing as you is difficult and awkward. It leads to moments where one party is upset because they can't do what they wanted to, or what they spawned in for.

I play PO for CAS. I know the feeling all too well when I've latejoined, only to see the Condor already in use. It sucks. I can't think of another role where you are _commonly unable to do the thing you're playing the game for_, and everybody just expects you to use the other option/sit with it. That's shit. That's not good design.

I've been on the opposite side of the coin, too. Watching the other pilot mope around all sad because I took the condor, and they wanted it.

At a minimum the roles should be seperated. We should have a designated Condor Pilot, and probably one for the tadpole too.

**Comment by Al-1ce** [comment]
<sub>ALTERNATIVELY: </sub>

The roles should be seperated. We should have a designated Condor Pilot, and one for the tadpole too. Fixes every problem I've seen discussed so far.

People are spawning as PO for _either_ the Tadpole or the Condor, not both. Reinforce the choice mechanically. Really easy solution.

**Comment by Al-1ce** [comment]
<sub>ALTERNATIVELY: </sub>

The roles should be seperated. We should have a designated Condor Pilot, and one for the tadpole too. Fixes every problem I've seen discussed so far.

**Comment by Al-1ce** [comment]
<sub>ALTERNATIVELY: </sub>

The roles should be seperated. We should have a designated Condor Pilot, and one for the tadpole too. Fixes every problem I've seen discussed so far.

90 hours as PO btw

**Comment by Al-1ce** [comment]
<sub>ALTERNATIVELY: </sub>

The roles should be seperated. We should have a designated Condor Pilot, and one for the tadpole too. Fixes every problem I've seen discussed so far.

**Comment by Naaanii** [comment]
> Even if synth, SL, CSE, or FC are using the tad, tad PO is still nice to have. Tad PO can stay on the tad to ensure it won't die, fighting off pressure with mounted hsg 102, or just taking off if too much pressure on it. Meanwhile, the synth, SL, CSE, or FC can leave the tad knowing its in good hands and that it won't be lost. I've worked with CSE Derrick a few times as tad PO. It's nice that he can go out and fix an APC, where he could easily be killed, while tad still remains safe under my supervision.
> 
> Not a fan of this change at all personally.

A regular marine/SL could do everything you've mentioned except CSE, synth or even a basic engineer with tad access would be better since they actually have engineering skills to help secure and defend the tad and aren't a role who's purpose is literally just having access.

**Comment by Naaanii** [comment]
> Non PO pilots have other things to do off the tad, PO should not.
> 
> I don't really see the point of this, if you want to do dedicated tad stuff, be a PO.

PO only exists for access and has nothing useful in terms of skills like engineering, medical or surgery. CAS at least makes a lot more sense for the role to exist while killing the bloat slot and letting other roles utilise it in ways the see fit seems like a better way to go about it unless the PO role itself gets changes (which I don't see happening).

**Comment by Naaanii** [comment]
> 

Dedicated PO is again a bloat role since its just Marine+ or SL- imo.

**Comment by Naaanii** [comment]
> Ignoring the fact that this would then divert important roles to have to play tad, leaving a hole where they left with nobody to fill it, this is pointless.
> 
> If a new player winds up on tad, sucks to suck currently and thats just how it is. Forcing PO's to play CAS just removes the dynamic part of their role that they can fufill different varities of duties in different ways. Even as some people treat PO as "the tad role", redirecting other important roles to play as "the tad role" (synth, SL, FC, etc) is using one puzzle piece to fill the hole of another: theres still a blank space left.

POs have the option to literally not pick the role if you don't want to CAS. This does not force redirected roles to use tad either as there are several rounds where tad goes unused. this merely just enables the option more.

**Comment by Losenis** [comment]
PO as a role is literally just an ahelp role to call dibs on something over others and be able to cry to admins if someone else gets there first, there's no actual benefit to going PO otherwise. It's gotten so bad that every time I see a PO take tad I just assume they're extremely bald, and 99% of the time they are, because a competent PO would instead go any other role for additional benefits and skills and ask for dropship access, like CSE, CMO, ST, normal engineer for 75 fucking vendor points to invest into the tad + its free gear, etc.

Because of this, many times Tadpole POs just suicide into xenos with nothing but an HSG and the cades wired due to just looting the metal from the cades around req and generally waste the tad when any other role is better equipped to run it. Inevitably this means just landing the tadpole behind marines, creating a giant choke, having the retreating marines all bunch up, and getting killed by boilers and praes.

**Comment by Starfirejordan** [comment]
people already explained why this PR is bad, but i come to say that this PR is bad

**Comment by Derrick2Electricboogaloo** [comment]
Maybe now I can use the tad without a late join PO coming down to steal if from me the moment I walk out of it for .2 nanoseconds instead of him silently waiting for me to drop my guard.

**Comment by Losenis** [comment]
Any competent PO will stop going PO if they want tadpole because every other role is better equipped for it, but a bald PO will nab the role and shit n piss over the tadpole being already taken by a non-PO. I've had some POs be so absolutely upset about the tadpole being taken that after like 50mins of the round having started they try to go groundside and steal the tadpole in the middle of a hold to the point they disobeyed captain orders and tried to get the admins to intervene.

Tadpole PO as a role has been obsolete since day 1. It's the actual worst role to use it for because everything from the other roles that you sacrifice for the access isn't even worth it because you can get access from CIC. The only thing that Tadpole PO as a role offers is enabling you to kick out the 3 engis about to dump 200+ points into the tadpole to put an HSG there and suicide into the xenos to the detriment of everyone else because the moment those 3 engis say anything about it you can ahelp it.

**Comment by Naaanii** [comment]
> moment when PO is expected to fly tadpole and has to beg for mats and a plasmacutter every round because they dont even have engineer skills to setup their own dropship. when's the last time you've seen a PO fly the Alamo either?

Not to mention with the removal of Engi vendor, they tend to steal sentries designed and balanced for FOB (unless ofc it's a tad op but you get the idea.)

**Comment by Al-1ce** [comment]
> Dedicated tad PO is again a bloat role since its just Marine+ or SL- imo.

I’m really confused at what you’re getting at here. How the hell can an exclusive condor pilot, *who stays shipside for the entirety of the round* function as a “Marine+”, or a bootleg SL? You’re being ridiculous here. 

The _entire point_ of separating the roles between pilots is to ensure that people are getting to play what they want. Spawning in as PO and having to pray that the other player didn't take the thing you wanted to use is dogshit game design and unfun for both parties.

>but CSE/ST/FC tadops?

CSE and ST are shitty roles in their current state and are unallowed to actually play the fucking game <sub> (shoot beno) </sub> without restrictions or getting in trouble. Until our current dated restrictions on either of these jobs are lessened or changed, I don't see it fair or even relevant to bring them up in regards to who gets to fly the tadpole. 

(Presumably the qualified pilot would be flying and not a fucking _architect_, but whatta I know?)

**Comment by FIXTHERADBOXPLEASE** [comment]
Just split PO into Tadpole and Condor pilots, and give the first one engineer-tier construction skills. It's not 'role bloat' if they both have clearly defined roles.

**Comment by MLGTASTICa** [comment]
Damn , can't even have the classic ritual of rite by fighting the other PO for the tad or condor now.

**Comment by FIXTHERADBOXPLEASE** [comment]
> > Just split PO into Tadpole and Condor pilots, and give the first one engineer-tier construction skills. It's not 'role bloat' if they both have clearly defined roles.
> 
> No.
> 
> #9589
That PR keeps PO as one role, and gives both engineering skill. I.E. it's not the same as what I suggested.

**Comment by Losenis** [comment]
>or vice versa

Dumb as brick POs tend to already do that when they inevitably lose the tadpole. The times they don't is because either they suicided so far into the xenos that nobody can recover them, or because they know the next PO will decap them with a t35.

>worse at reinforcing SD

Dude I cryo every time it goes to hijack. Is this the next thing you're going to make shit up about?

**Comment by FIXTHERADBOXPLEASE** [comment]
A Condor Pilot with no engineering skill will logically be worse at reinforcing SD than a Tadpole Pilot with 1 engineering skill.
That's the only time I can imagine one Pilot being objectively better than the other, and it's when both of their aircraft are out of use. One isn't "much better than the other."

**Comment by FosterGrime-Heymynameslouie** [comment]
yeah! maybe a PO role specialized for cas and a PO role specialized for tadpole, like PO Tad Crew or PO Close Air Support would be better so you can avoid scenrious where po's do a blood battle between who gets who (still awsome to watch them fight thoug) and would keep the normal command roles ie fc's/sl's/synths from doing their own jobs just so they can man tadpole, when 2 po's before could already do that just well

they dont even have to have a change in their skill set, just a seperation between cas and tad

**Comment by Naaanii** [comment]
> > > Just split PO into Tadpole and Condor pilots, and give the first one engineer-tier construction skills. It's not 'role bloat' if they both have clearly defined roles.
> > 
> > 
> > No.
> > #9589
> 
> That PR keeps PO as one role, and gives both of them engineering skill. I.E. it's not the same as what I suggested. EDIT: As a side note, that would also stop POs from killing eachother/themselves when they don't get their preferred vessel. Which is still a once-a-week occurrence minimum.

Tbf, removing the extra slot would also stop POs from killing each other and other roles can't really fight for it either as we already have the chain of command preventing overrules or tad theft. Late join POs, especially if they're a tad focused PO, can still late join and technically steal tad as they have no indication if it's being used or not before joining and well, splitting the roles wouldn't completely solve the issue either. SLs/FCs/CSEs/Synths can all go about their business and focus on other things whether the tad is taken or not so there's more flexibility in giving more leeway to them rather than having a sole role which isn't even solely access locked to it. (Making the access stricter would be even worse design).

**Comment by Naaanii** [comment]
> yeah! maybe a PO role specialized for cas and a PO role specialized for tadpole, like PO Tad Crew or PO Close Air Support would be better so you can avoid scenrious where po's do a blood battle between who gets who (still awsome to watch them fight thoug) and would keep the normal command roles ie fc's/sl's/synths from doing their own jobs just so they can man tadpole, when 2 po's before could already do that just well
> 
> they dont even have to have a change in their skill set, just a seperation between cas and tad

See above as this does not solve the late join issue either.

**Comment by Al-1ce** [comment]
> removing the extra slot would also stop POs from killing each other 

Tadpole being taken will still occur with FC/Synth/CSE.

> and other roles can't really fight for it either as we already have the chain of command preventing overrules or tad theft

Amusing if you actually believe this. 

>latejoin issue

This wouldn't exist if the jobs were specified for their purpose, which you would expect to be the case...

### I'll reiterate, again: 
Splitting the roles would fix nigh every problem brought up here. The community widely seems to agree this is the better solution. But you're hellbent on making that not the case. Why?

**Comment by Losenis** [comment]
>Amusing if you actually believe this.

I've gotten bwoinked over it. I've killed POs over it. I've had to call Cap'n Gramps in over it. I've had a TED talk in the policy channel in the discord server over it. It happens. Rarely, but it happens.

**Comment by FIXTHERADBOXPLEASE** [comment]
That sounds like a personal problem if it only happens with you, Tanya.

Anyway, yeah. 
When I said 'stop POs from killing eachother/themselves,' I was comparing it to baseline. Having only one PO sorta stops it too, of course.
Only sort of. 
Removing a PO slot makes the CSE the dedicated Tadpole job, with the sole difference that he's not allowed to stay groundside if the Tadpole is destroyed. Both POs and CSEs can't stray from the Tadpole while it's intact.
But there's nothing stopping the Pilot from deciding he wants to use the Tadpole over the Condor - it even still mentions the Tadpole in the blurb IIRC - which just makes the same issue. 
Unless it's specified that Pilots, as a role, are Condor/Alamo(lmao) only, and that they're not supposed to touch the other important aircraft in the hangar.

The thing is, if CSE/ST is going to become the dedicated Tadpole job, as would happen with this PR, why not just split the PO into one job for each aircraft? 
There's very little difference between the two options now that I think about it. It's just more slightly less intuitive for everyone else that a "Ship Technician" Or "Ship Engineer" flies an aircraft instead of a "Pilot." I mean, I GUESS it's a ship?
But when you KNOW a role is meant for a certain thing, there's no confusion going into it. If the respective slot's gone, you're not flying the Condor/Tadpole.

Plus, having a Tadpole Pilot would let the job be further specialized for the Tadpole. Good for futureproofing.

**Comment by FIXTHERADBOXPLEASE** [comment]
Apologies for being long-winded, my Internet is down so I can't play the server, only post on TGMC github

**Comment by Naaanii** [comment]
> That sounds like a personal problem if it only happens with you, Tanya.
> 
> Anyway, yeah. When I said 'stop POs from killing eachother/themselves,' I was comparing it to baseline. Having only one PO sorta stops it too, of course.
> 
> Only sort of.
> 
> Removing a PO slot makes the CSE the dedicated Tadpole job, with the sole difference that he's not allowed to stay groundside if the Tadpole is destroyed. Both POs and CSEs can't stray from the Tadpole while it's intact.
> 
> But there's nothing stopping the Pilot from deciding he wants to use the Tadpole over the Condor - it even still mentions the Tadpole in the blurb IIRC - which just makes the same issue.
> 
> Unless it's specified that Pilots, as a role, are Condor/Alamo(lmao) only, and that they're not supposed to touch the other important aircraft in the hangar.
> 
> The thing is, if CSE/ST is going to become the dedicated Tadpole job, as would happen with this PR, why not just split the PO into one job for each aircraft?
> 
> There's very little difference between the two options now that I think about it. It's just more slightly less intuitive for everyone else that a "Ship Technician" Or "Ship Engineer" flies an aircraft instead of a "Pilot." I mean, I GUESS it's a ship?
> 
> But when you KNOW a role is meant for a certain thing, there's no confusion going into it. If the respective slot's gone, you're not flying the Condor/Tadpole.
> 
> Plus, having a Tadpole Pilot would let the job be further specialized for the Tadpole. Good for futureproofing.

CSE/ST never has or never will be a 'dedicated' Tad job. They are free to work/stay shipside, handle req or even reinforce the FOB groundside.

Once again, removing the slot allows for more interesting uses for the tad except now a bloat role isn't confined to it. SLs, FCs, Synths (and on rare occasions) even CMOs can use the tad to support the marines in different ways.

Giving a dedicated Tad role engi skills would probably be ideal but again, it was denied and once again this doesn't solve the latejoin issue as most leadership roles already have TAD access and restricting said access would just be worse imo.

**Comment by BettaSplendid** [comment]
While thumbs aren't a strong judgment of quality, this PR is unpopular and has little impact. I don't think it is worth forcing it through for how few benefits it may provide. 

Personally, I think dedicated tad & condor roles are the better way, if we change road at all.

**Comment by Aporhtonoma** [comment]
due to how many pilots seen to be bald after coming in second and dont get to fly the big metal jet so they try their luck with tad(and some cursed times manual alamo)  I agree with the pr but we might as well rename pilot officer to condor pilot at this point to avoid confusion since taking the only role who can fly cas to NOT use cas seems stupid.

**Comment by Losenis** [comment]
All tadpole POs are bald because if you're competent you'll pick any other job and then ask for tadpole access if you don't have it already. I am not exaggerating, almost every single tadpole PO ends up with either the tadpole dead on some random spot on the map or landing somewhere that causes almost grief levels of damage to marines. The only competent POs are those that are metaganging with actual proper jobs to grab the tadpole as a group, which are only just going PO for the dibs and ahelping if anyone wants to steal it than any actual ingame incentive to go that job.

**Comment by KerbalCitizen** [comment]
> due to how many pilots seen to be bald after coming in second and dont get to fly the big metal jet so they try their luck with tad(and some cursed times manual alamo) I agree with the pr but we might as well rename pilot officer to condor pilot at this point to avoid confusion since taking the only role who can fly cas just to NOT use cas seems stupid.

I believe CAS only requires access to fly, although powerloader skill helps with rearming after flying. I've heard of captain and staff officer flying CAS before, but it is fairly rare to see.

**Comment by Al-1ce** [comment]
> All tadpole POs are bald because if you're competent you'll pick any other job and then ask for tadpole access if you don't have it already. I am not exaggerating, almost every single tadpole PO ends up with either the tadpole dead on some random spot on the map or landing somewhere that causes almost grief levels of damage to marines. The only competent POs are those that are metaganging with actual proper jobs to grab the tadpole as a group, which are only just going PO for the dibs and ahelping if anyone wants to steal it than any actual ingame incentive to go that job.


This really hits home.
So, how about an alternative to all of this..

Make the PO _only_ able to fly the condor. Leave the tadpole to FC/Synth/whateverthefuck else. Bit unorthodox but I think it'd solve the issue of latejoins/other roles wanting to go tad.

**Comment by Naaanii** [comment]
> > All tadpole POs are bald because if you're competent you'll pick any other job and then ask for tadpole access if you don't have it already. I am not exaggerating, almost every single tadpole PO ends up with either the tadpole dead on some random spot on the map or landing somewhere that causes almost grief levels of damage to marines. The only competent POs are those that are metaganging with actual proper jobs to grab the tadpole as a group, which are only just going PO for the dibs and ahelping if anyone wants to steal it than any actual ingame incentive to go that job.
> 
> This really hits home. So, how about an alternative to all of this..
> 
> Make the PO _only_ able to fly the condor. Leave the tadpole to FC/Synth/whateverthefuck else. Bit unorthodox but I think it'd solve the issue of latejoins/other roles wanting to go tad.

Hence the removal of the slot. I could probably do access as well or instead just name it CAS pilot and leave the access for emergency scenarios.

**Comment by Al-1ce** [comment]
> Hence the removal of the slot. I could probably do access as well or instead just name it CAS pilot and leave the access for emergency scenarios.

I actually like this. Not the emergency scenario thing since the round is basically over by then, but making the PO an exclusive CAS pilot with 1 slot fixes the latejoin issue and allows other roles (FC, CSE, Synth,) to tadop without a problem.

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Do not whipe hours

**Comment by breadhunt** [comment]
CSE, ST + synth are all roles *designed to fill in for duties.* No shit they can take the tadpole when there's not a PO there.  It wouldn't make sense to remove RO for the same reason. 
These roles are also often shafted into doing req, shipside medbay or CIC+engineering when there's nobody else available.
There also isn't always people to provide access for tadpole - only CIC/CMO/CSE/AI/FC can do it. 
A tadpole dropship role would provide *guaranteed access to tadpole*, and that's what the role **SHOULD** be.

But latejoins exist! If CSE, ST, synth DOES take tadpole - the role is useless, because the tadpole is busy bitch. 
I think the best solution would be to add a "claim" button on the tadpole which disables the tadpole pilot role (which should be possible codewise).

This means the tadpole pilot role would actually fill a niche - it's not just "ahelp rights to tadpole", but actual, MECHANICAL rights to the tadpole. If that big, shiny PO role is open, you can be sure that you'll get to play with the tadpole. 

As this PR stands, getting tad will never be guaranteed. Every single roundstart role here has a duty of responsibility that can prevent you from taking the tadpole if you want it. And fhat's a shame - I've been playing some tad PO recently and it is is fucking *fun*. Only just noticed this PR now, guh.

PO is bloat and there's better alternatives? Add a separate tad pilot role as the other suggested. @Naaanii your PR applied to both tad and condor. It also didn't explain the real issues with PO discussed here - the PR body, paraphrasing, says "i would like it if PO had better engineering skills so that it is a more powerful role". But you didn't say how you want it to be a more powerful role *because it's currently being overshadowed by literally every other role that could take it*. Doublecheck with maints, but that's not powercreep, that's carving out a niche for what you have described as just a noobtrap, and that's much better design than rolling CSE or whatever sometimes just being denied the ability to play with one of the coolest fucking features we have.

--

Separate the roles into condor pilot and tadpole pilot, that alone is a massive fucking QoL which is so, so, so worth losing silly 2d spessman playtime hours for. Add a big "ONE-USE CLAIM" button to the tadpole/a new console just outside it (maybe with a big confirm box? maybe which closes tadpole shutters until it's pressed so people remember to claim it ASAP?) that can only be used by people with fancy tadpole access (to prevent griffing) and that disables PO slot when it's pressed. If you're feeling it, give tad PO some extra skills. But they might not even need them; just having a role which actually *guarantees* the use of tadpole not just on paper but in practice is good enough.

TL;DR there are alternatives to this. Besides I don't see how this actually fixes your problem - a bald synth on tadpole is just as bad as a bald PO, if not worse because they can't use any deployable MGs.

**Comment by Naaanii** [comment]
> CSE, ST + synth are all roles _designed to fill in for duties._ No shit they can take the tadpole when there's not a PO there. It wouldn't make sense to remove RO for the same reason. These roles are also often shafted into doing req, shipside medbay or CIC+engineering when there's nobody else available. There also isn't always people to provide access for tadpole - only CIC/CMO/CSE/AI/FC can do it. A tadpole dropship role would provide _guaranteed access to tadpole_, and that's what the role **SHOULD** be.
> 
> But latejoins exist! If CSE, ST, synth DOES take tadpole - the role is useless, because the tadpole is busy bitch. I think the best solution would be to add a "claim" button on the tadpole which disables the tadpole pilot role (which should be possible codewise).
> 
> This means the tadpole pilot role would actually fill a niche - it's not just "ahelp rights to tadpole", but actual, MECHANICAL rights to the tadpole. If that big, shiny PO role is open, you can be sure that you'll get to play with the tadpole.
> 
> As this PR stands, getting tad will never be guaranteed. Every single roundstart role here has a duty of responsibility that can prevent you from taking the tadpole if you want it. And fhat's a shame - I've been playing some tad PO recently and it is is fucking _fun_. Only just noticed this PR now, guh.
> 
> PO is bloat and there's better alternatives? Add a separate tad pilot role as the other suggested. @Naaanii your PR applied to both tad and condor. It also didn't explain the real issues with PO discussed here - the PR body, paraphrasing, says "i would like it if PO had better engineering skills so that it is a more powerful role". But you didn't say how you want it to be a more powerful role _because it's currently being overshadowed by literally every other role that could take it_. Doublecheck with maints, but that's not powercreep, that's carving out a niche for what you have described as just a noobtrap, and that's much better design than rolling CSE or whatever sometimes just being denied the ability to play with one of the coolest fucking features we have.
> 
> --
> 
> Separate the roles into condor pilot and tadpole pilot, that alone is a massive fucking QoL which is so, so, so worth losing silly 2d spessman playtime hours for. Add a big "ONE-USE CLAIM" button to the tadpole/a new console just outside it (maybe with a big confirm box? maybe which closes tadpole shutters until it's pressed so people remember to claim it ASAP?) that can only be used by people with fancy tadpole access (to prevent griffing) and that disables PO slot when it's pressed. If you're feeling it, give tad PO some extra skills. But they might not even need them; just having a role which actually _guarantees_ the use of tadpole not just on paper but in practice is good enough.
> 
> TL;DR there are alternatives to this. Besides I don't see how this actually fixes your problem - a bald synth on tadpole is just as bad as a bald PO, if not worse because they can't use any deployable MGs.

Separate PO roles don't solve the issue as mentioned above.

**Comment by Naaanii** [comment]
> Do not whipe hours

Fair

**Comment by breadhunt** [comment]
> Separate PO roles don't solve the issue as mentioned above.

You're right, not on their own they don't. But again, removing the PO slot *only removes the ability to be able to reliably take the tadpole* and this could cause even more issues if people roll more important roles than PO for it (e.g synth, FC) and refuse to do other vital duties. 

The niche of the PO role **should** be guaranteed access to tadpole. The issues you've talked about regarding tadpole POs - it being an "ahelp role" that is only used to causs fighting over the tadpole - would be resolved through mechanical changes, e.g a claim button for people with tadpole access so they can reserve the tadpole.

And again, although splitting the slots into two roles wouldn't magically fix PO being weaker than CSE/synth/whatever, *making tad PO into a separate role allows us to bring it on par with alternatives without powercreeping condor PO*. Unless I'm missing something neither of these points have been addressed above

**Comment by Lumipharon** [comment]
Who is using the tad is an IC issue, and people should be reasonable enough to sort it out IC without reeing at each other or blowing each other up.

If people are taking tad with other roles because they have mechanic advantages to powergame in such a way, I am more inclined to remove their access to the tad, as opposed to removing a PO to encourage it.

**Comment by Losenis** [comment]
There is literally no incentivze whatsoever to take tadpole as a PO than any other job other than crying in ahelp that another job took it..

**Comment by Al-1ce** [comment]
> Who is using the tad is an IC issue

It becomes an OOC issue because it's terrible game design. There is no other role on the server where you aren't guaranteed your job. I.E, the thing you want to do.

When the PO spawns and whatever they want is taken? Tough luck. Imagine if we treated other roles the same way. Picture spawning in as a regular squad marine to see every gun and vendor empty. You'd either wait and scavenge around, or cryo. 

That is *exactly* how the PO feels.
